### DISCUSSION TITLE
cuarto-blog strange behavior when using subtitle instead of description in the yaml of a post

### DISCUSSION THREAD

**Discussion by ManuelSpinola** [quarto-dev/quarto-cli#752] [Q&A]
quarto-blog: I noticed when using subtitle (instead of description) in the yaml of the post that the rendered website the title of the post appears with characters like {r} and html characters.

I cannot reproduce this with all my qmd files in posts but it happens with some of them.  Even the rendered files looks as expected but the website not.

How can I solve this problem?

**Comment by dragonstyle** [comment]
Can you share an example qmd file that exhibits this behavior? Thanks!

**Comment by ManuelSpinola** [comment]
Yes of course.

How can I share the qmd file?

These are the other files necessary to run the qmd

[metricas_1km.xlsx](https://github.com/quarto-dev/quarto-cli/files/8583895/metricas_1km.xlsx)
[metricas_8km.xlsx](https://github.com/quarto-dev/quarto-cli/files/8583896/metricas_8km.xlsx)
![preview](https://user-images.githubusercontent.com/22131335/165782641-947b923a-b2f1-4df4-b431-1e7145aee32a.png)
.

**Comment by dragonstyle** [comment]
You can just point me at a link if it is somewhere on Github, otherwise you can just post it here or email it to me (my first name- charles at rstudio.com). thx!

**Comment by dragonstyle** [comment]
Ok, @ManuelSpinola shared a `qmd` with me offline. The basic issue was how we were inferring descriptions for items when there wasn't an explicit `description` or `abstract`. In the case neither of these properties is present in the document front matter, we look in the document itself to craft a description. We were doing this using the raw markdown (and regexes) and as you'd expect this was rife with issues (for example if code blocks appeared very early in the document). I update our approach to match the way we handle feed content, allowing the content to be fully rendered to HTML, then using the rendered HTML to update the listing description. This should resolve the issue and allow the description to become (truly) the first paragraph of content in the document.

The fix is here: 1d18844fa0e16af5f350bd006b5479f03cdebe6b and a fresh release should be ready in a few minutes. Thanks for reporting this!
